dosagelib/plugins/t.py

```diff
@@ -188,6 +189,33 @@ def namer(self, image_url, page_url):
         return filename
 
 
+class Turnoff(_ParserScraper):
+    name = 'turnoff'
+    url = 'https://turnoff.us/'
+    imageSearch = '//article[%s]//img' % xpath_class('post-content')
+    prevSearch = '//div[%s]//a' % xpath_class('prev')
+    stripUrl = url + 'geek/%s'
+    firstStripUrl = stripUrl % 'tcp-buddies'
+    multipleImagesPerStrip = True
+
+    def __init__(self, *args, **kwargs):
+        super(Turnoff, self).__init__(*args, **kwargs)
+        # Neither the images nor the pages contain information about dates or indices.
+        # However we can extract the order of the images from the JavaScript.
+        html = requests.get(self.url).text
```

Comment:
1. Please use the existing requests session instead (self.session IIRC)
2. Don't execute any HTTP requests in the `__init__` method of a scraper. This will run far too often (even if just listing all available plugins). Maybe just make `comics_order` a `@property` with a `@lru_cache()`?